src/main/java/org/opensearch/ad/transport/AnomalyResultTransportAction.java

```diff
@@ -1145,10 +1194,14 @@ public void onFailure(Exception e) {
         private void handleEntityResponses() {
             if (failure.get() != null) {
                 listener.onFailure(failure.get());
-            } else if (ackResponses.isEmpty()) {
+            } else if (entityResultResponses.isEmpty()) {
                 listener.onFailure(new InternalFailure(adID, NO_ACK_ERR));
             } else {
-                listener.onResponse(new AnomalyResultResponse(0, 0, 0, new ArrayList<FeatureData>()));
+                long maxTotalUpdates = 0;
+                while (!entityResultResponses.isEmpty()) {
+                    maxTotalUpdates = Math.max(maxTotalUpdates, entityResultResponses.poll().getTotalUpdates());
+                }
+                listener.onResponse(new AnomalyResultResponse(0, 0, 0, new ArrayList<FeatureData>(), maxTotalUpdates, intervalMinutes));
```

Comment:
This may require a rewrite as current main branch has different logic.

Reply:
Thanks for reminding. Will resolve in separate PR.